android/src/main/kotlin/com/gdelataillade/alarm/notification/NotificationOnKillService.kt

```diff
@@ -70,9 +78,9 @@ class NotificationOnKillService : Service() {
             val notificationManager: NotificationManager =
                 getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
             notificationManager.createNotificationChannel(channel)
-            notificationManager.notify(NOTIFICATION_ID, notificationBuilder.build())
+            startForeground(NOTIFICATION_ID, notificationBuilder.build())
```

Comment:
What is the difference between this and ```notificationManager.notify```.
Still sometimes it shows this notification, and sometimes doesn't

Reply:
an attempt...

Reply:
Imho it doesn't make sense add this in current form. Let's leave this for later as it is not working, and only shows the notification sometimes when it is removed from recents. Alarms will work in that case anyway.